### DISCUSSION TITLE
internal wiring front port to front port

### DISCUSSION THREAD

**Discussion by Cableaktive** [netbox-community/netbox#15071] [Q&A]
Hi,

Usually my front ports are connected to rear ports, in internal wiring of the device/module.

But today I have a case of a module where the front ports are connected to other front ports (in internal).

I can't because a front port must be connected to a rear port.

How to do this, please ?

I can create fake rear ports wired together, and tag these fake ports and fake cables as fake but it's not great

Chatgpt tells me about virtual ports and virtual cables to do that, but I don't believe this concept exists in netbox

Thanks

**Comment by candlerb** [comment]
The only other way I know is to create some "rear ports" and treat them as being on the front.

I had to do this before, where I created front ports 1,3,5,7... and rear ports 2,4,6,8... (in an Openreach EAD device, where port 1 is a passthrough to port 2, etc). These are all physically on the front, but Netbox doesn't show front and rear panel layouts anyway.
